.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I added [Prettier](https://prettier.io/) mainly for writing jsdoc comments. It does nice things like wrapping documentation text. I think it's also a nice tool for largely removing code style from the list of things to worry about.